README.md

```diff
@@ -50,6 +50,33 @@ $ pip install -r requirements-dev.txt
 
 Then use ``tox`` to run the complete suite against the full set of build targets, or ``pytest`` to run tests against a specific version of Python. If you are using ``pytest`` you must include the ``-p no:pytest-brownie`` flag to prevent it from loading the Brownie plugin.
 
+### Using Docker
+
+You can use a sandbox container provided in the docker-compose.yml file for testing inside a Docker environment.repository
+
+This container provides everything you need to test using a Python 3.6 interpreter.
+
+Start the test environment:
+
+```bash
+docker-compose up -d
+```
+
+To open a session to the container:
+```bash
+docker-coampose exec sandbox bash
+```
+
+To run arbitrary commands, use the `bash -c` prefix.
+```bash
+docker-compose exec sandbox bash -c ''
+```
+
+For example, to run the tests in `brownie/tests/test_format_input.py`:
+```bash
+docker-compose exec sandbox bash -c 'python -m pytest tests/convert/test_format_input.py'
```

Comment:
Why should we suggest using compose when it is one container and will likely always be just one container?  Maybe docker run / docker exec would be more direct?

Reply:
I was debating the same thing myself.  First, this is how web3.py organizes it so it seemed reasonable to provide a similar development experience.

Secondly, maybe this is just my naivety on Docker, but isn't using docker-compose one less command for the user? Here it's just `docker-compose up -d` then `docker exec`. Otherwise, it would be `docker build`, `docker run`, then `docker exec`? docker-compose also removes the need to remember to bind mount the working directory.

I'm fine removing docker-compose if you don't agree with those tradeoffs - let me know.

Reply:
You are correct in that it removes a few steps; I'm fine with leaving it. As a docker power user, you could look at the compose and decide that you just want to build/run/exec it yourself.